apps/frontend/src/modules/user-courses/libs/types/user-courses-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { type UserCourseDto } from "./user-course-dto.type.js";
+
+type UserCoursesResponseDto = {
+	courses: UserCourseDto[];
```

Comment:
```suggestion
	items: UserCourseDto[];
```
add to to shared folder. use on both backend/frontned